docs/advanced/java-client.md

```diff
@@ -136,6 +110,71 @@ assertThat(graphQLResponse.errors.get(0).extensions.errorType).isEqualTo(ErrorTy
 assertThat(graphQLResponse.errors.get(0).extensions.errorDetail).isEqualTo("FIELD_NOT_FOUND")
 ```
 
+
+## Plug in your own HTTP client
+
+Instead of using WebClient, you can also plug in your own HTTP client.
+This is useful if you already have a configured client for your backend with authN/authZ, TLS, etc.
+In this case you are responsible for making the actual request, but the GraphQL client wraps the HTTP client and provides easy parsing of GraphQL responses.
+
+There are two interfaces that you can pick from:
+
+* GraphQLClient: For blocking HTTP clients
+* MonoGraphQLClient: For non-blocking HTTP clients
+
```

Comment:
Call out explicitly that the `CustomGraphQLClient` and `CustomMonoGraphQLClient` are implementations of the above interfaces?